docs/Configuration/yaml-files.md

```diff
@@ -487,6 +495,7 @@ Use `labels_include_any` to target hosts that have any label or `labels_exclude_
 - `uninstall_script.path` is the script Fleet will run on hosts to uninstall software. The [default script](https://github.com/fleetdm/fleet/tree/main/pkg/file/scripts) is dependent on the software type (i.e. .pkg).
 - `post_install_script.path` is the script Fleet will run on hosts after the software install. There is no default.
 - `self_service` specifies whether or not end users can install from **Fleet Desktop > Self-service**.
+- `categories` is an array of categories. See supported categories [here](#example7).
```

Comment:
@noahtalerman Relying on "the 7th Example header" will cause links to silently drift. Seems like it'd be better to add `#### Supported software categories` above the list and link to there.

Reply:
Revised. Also swapped "here" for a semantically more meaningful link title as "click here" and the like is poor a11y.